src/argilla/client/feedback/schemas/validators.py

```diff
@@ -45,3 +45,17 @@ def validate_numeric_metadata_property_bounds(cls: "BaseModel", values: Dict[str
         raise ValueError(f"`{cls.__name__}` with name={name} cannot have `max` less than `min`")
 
     return values
+
+
+def validate_numeric_metadata_filter_bounds(cls: "BaseModel", values: Dict[str, Any]) -> Dict[str, Any]:
+    name = values.get("name")
+    le = values.get("le")
+    ge = values.get("ge")
+
+    if le is None and ge is None:
+        raise ValueError(f"`{cls.__name__}` with name={name} must have at least one of `le` or `ge`")
+
+    if le is not None and ge is not None and le >= ge:
```

Comment:
If not, we cannot support filtering records with just one value in a numeric metadata property. For example: fetching records with `metadata.value == 10`. We can define in terms of filter as : `le=10, ge=10`
```suggestion
    if le is not None and ge is not None and le > ge:
```